x-pack/test_serverless/api_integration/test_suites/common/security/user_profiles.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default function ({ getService }: FtrProviderContext) {
+  const svlCommonApi = getService('svlCommonApi');
+  const supertest = getService('supertest');
+
+  describe('security/user_profiles', function () {
+    describe('route access', () => {
+      describe('disabled', () => {
```

Comment:
__note:__ As I mentioned above, we need user profiles APIs in Serverless, so we'll likely change these API tests to test "internal" APIs instead.